ui/static/ui/js/listing.js

```diff
@@ -1,6 +1,78 @@
 define(['jquery', 'bootstrap', 'icheck', 'csrf'], function(jQuery) {
   'use strict';
 
+  /* global $ */
+
+  function formatGroupName(string) {
+    string = string.charAt(0).toUpperCase() + string.slice(1);
+    return string.substring(0, string.length - 1);
+  }
+
+  /* This is going to grow up to check whether
+   * the name is a valid Kerberos account
+   */
+  function isMITEmail(kerberosId) {
+    var email = kerberosId + '@mit.edu';
+    var regex = /^([a-zA-Z0-9_.+-])+\@(([a-zA-Z0-9-])+\.)+([a-zA-Z0-9]{2,4})+$/;
```

Comment:
this isn't the right regex to validate an email address (http://emailregex.com/)